script.js

```diff
@@ -1,4 +1,152 @@
+    'Now you can type "scissors" "paper" or "stone" to play another round!';
+
+  //scoring system
+  var scoreUser = 0;
+  var scoreProgramme = 0;
+  var draw = 0;
+
+  //user won
+  if (
+    (userInput == "Scissors" && programmeInput == "Paper") ||
+    (userInput == "Stone" && programmeInput == "Scissors") ||
+    (userInput == "Paper" && programmeInput == "Stone") ||
+    (userInput == "Reversed Scissors" && programmeInpupt == "Reversed Stone") ||
+    (userInput == "Reversed Paper" && programmeInput == "Reversed Scissors") ||
+    (userInput == "Reversed Stone" && programmeInput == "Reversed Paper")
+  ) {
+    scoreUser = scoreUser + 1;
+    return (
+      "You won! you chose " +
+      userInput +
+      ". " +
+      "<br><br>" +
+      "The programme chose " +
+      programmeInput +
+      "." +
+      "<br><br>" +
+      "your score = " +
+      scoreUser +
+      "<br><br>" +
+      "programme has scored = " +
+      scoreProgramme +
+      "<br><br>" +
+      "Both User and Programme draw =" +
+      draw +
+      "<br><br>" +
+      instructionsOnceGameCompletes
+    );
+  }
+
+  //program won
+  else if (
+    (userInput == "Paper" && programmeInput == "Scissors") ||
+    (userInput == "Scissors" && programmeInput == "Stone") ||
+    (userInput == "Stone" && programmeInput == "Paper") ||
+    (userInput == "Reversed Stone" && programmeInpupt == "Reversed Scissors") ||
+    (userInput == "Reversed Scissors" && programmeInput == "Reversed Paper") ||
+    (userInput == "Reversed Paper" && programmeInput == "Reversed Stone")
+  ) {
+    scoreProgramme = scoreProgramme + 1;
+    return (
+      "you lose! " +
+      "<br><br>" +
+      "Bummer! " +
+      "<br><br>" +
+      "you chose " +
+      userInput +
+      ". " +
+      "<br><br>" +
+      "The computer chose " +
+      programmeInput +
+      "." +
+      "<br><br>" +
+      "your score = " +
+      scoreUser +
+      "<br><br>" +
+      "programme has scored = " +
+      scoreProgramme +
+      "<br><br>" +
+      "Both User and Programme draw =" +
+      draw +
+      "<br><br>" +
+      instructionsOnceGameCompletes
+    );
+  }
+
+  //draw - user and programme same input
+  else {
+    draw = draw + 1;
+    return (
+      "it's a draw." +
+      "<br><br>" +
+      "you chose " +
+      userInput +
+      ". " +
+      "<br><br>" +
+      "The computer chose " +
+      programmeInput +
+      "." +
+      "<br><br>" +
+      "your score = " +
+      scoreUser +
+      "<br><br>" +
+      "programme has scored = " +
+      scoreProgramme +
+      "<br><br>" +
+      "Both User and Programme draw =" +
+      draw +
+      "<br><br>" +
+      instructionsOnceGameCompletes
+    );
+  }
+};
+
+//username
+var currentGameMode = "Waiting for Username.";
+var userName = input;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //user to put in username
+  if (currentGameMode == "waiting for user name") {
```

Comment:
it may not be working because of the difference in capitalisation betwen this and line 123
can consider using scream case var for the string to avoid this error of replication.

var USERNAMEMODE = "Waiting for Username.";
var currentGameMode = USERNAMEMODE 
if (currentGameMode == USERNAMEMODE)